<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1456</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="loginTextEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>My Profile</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendsTable">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>251</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="suggestFriendsTable">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>100</y>
      <width>261</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QLabel" name="recentPostsLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>971</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="passwordTextEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="loginPasswordLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your password: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="toSignUpButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>0</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToLogInButton">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back To Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signUpButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>420</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Sign Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchBarLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search and add your first friend to initialize your page!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="searchBarTextEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBarButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search and Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginMainLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>Login Page</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordSignUpLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Set your password: </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="passwordSignUpTextEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="searchFriendLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1456</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
